notebooks/05-operations/02-druid-logs-asynch.ipynb

```diff
@@ -0,0 +1,381 @@
+    "\n",
+    "In the `middleManager` log, notice:\n",
+    "\n",
+    "* The creation of Task Runners, each with their own Java configurations.\n",
+    "* `FileTaskLogs` writing logs to specific folders for each Task Runner.\n",
+    "* Tasks being recorded as completed.\n",
+    "\n",
+    "Press `q` to exit the panel, then hit `b` and open the Historical log.\n",
+    "\n",
+    "Notice how the Historical process sees the segments and loads these from Deep Storage (being in `var/druid/segments` in this environment) and into its own local Segment Metadata cache (being in `var/druid/segment-cache`).\n",
+    "\n",
+    "These logs therefore give you important diagnostic information about Druid's ingestion service:\n",
+    "\n",
+    "* Ingestion planning at the Overlord.\n",
+    "* Parallel ingestion task creation and execution from the Overlord to the Middle Manager.\n",
+    "* Segment publishing by the Overlord after task completion.\n",
+    "* Data distribution by the Coordinator and Historicals.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b680cce2-652c-447c-b96e-6be0bc61ab54",
+   "metadata": {},
+   "source": [
+    "## Task logs\n",
+    "\n",
+    "In the example above, the MiddleManager spawned multiple other tasks (peons) that created logs then [written to a log location](https://druid.apache.org/docs/latest/ingestion/tasks#task-logs) that you can configure as an administrator.\n",
+    "\n",
+    "Run this command to view the `common.runtime.properties` for the `start-druid` environment.\n",
+    "\n",
+    "```bash\n",
+    "cat ~/learn-druid-local/apache-druid/conf/druid/auto/_common/common.runtime.properties\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28c0f9d0-9127-4d2c-a97e-d205fafae862",
+   "metadata": {},
+   "source": [
+    "### Viewing local logs\n",
+    "\n",
+    "The \"Indexing Service Logs\" section contains the configuration that caused the Middle Manager to write its logs to local disk:\n",
+    "\n",
+    "```\n",
+    "druid.indexer.logs.type=file\n",
+    "druid.indexer.logs.directory=var/druid/indexing-logs\n",
+    "```\n",
+    "\n",
+    "Run this command to see where task logs were written by looking at the Middle Manager log:\n",
+    "\n",
+    "```bash\n",
+    "grep \"Wrote task log to\" middleManager.log\n",
+    "```\n",
+    "\n",
+    "Notice that the log filename is made up of the query ID and the suffix `.log`.\n",
+    "\n",
+    "Adapt the following `vi` command to open the log for viewing:\n",
+    "\n",
+    "```bash\n",
+    "vi ~/learn-druid-local/apache-druid/<logfilelocation>\n",
+    "```\n",
+    "\n",
+    "For example,\n",
+    "\n",
+    "```bash\n",
+    "vi ~/learn-druid-local/apache-druid/var/druid/indexing-logsquery-83651faf-d179-4f4d-b0ad-ce61e36e85bd-worker0_0.log\n",
+    "```\n",
+    "\n",
+    "The logs contain a large amount of information helpful in understanding what Druid does during an ingestion. \n",
```

Comment:
I think it would be good to provide examples of each of these types of log entries in the task log.